pkg/build/builder/docker.go

```diff
@@ -36,6 +37,17 @@ const (
 	noOutputDefaultTag = "no_repo/no_output_default_tag"
 )
 
+const (
+	// Label represents the LABEL Docker instruction
+	Label MetaInstuction = "LABEL"
+	// Env represents the ENV Docker instruction
+	Env MetaInstuction = "ENV"
+)
+
+// MetaInstuction represent an Docker instruction used for adding metadata
+// to Dockerfile
+type MetaInstuction string
```

Comment:
It's slightly more logical to have the type def first and only afterwards the constants.